cmd/sync.go

```diff
@@ -42,6 +45,18 @@ to get Kong's state in sync with the input state.`,
 		return nil
 	},
 	PreRunE: func(cmd *cobra.Command, args []string) error {
+		if syncCmdKongStateFileFromStdin == true {
+			syncCmdKongStateFile = "-"
+		} else {
+			// cobra doesn't seem to allow managing the "--" or "-"
+			//   flag directly so it has to be parsed out of os.Args
+			for i, arg := range os.Args {
+				if i > 1 && arg == "--" {
+					syncCmdKongStateFile = "-"
+					break
+				}
+			}
+		}
```

Comment:
This is also unnecessary and merely treating `-` as a special value to the `--state` arg should be for this feature.